test/common/dns.js

```diff
@@ -287,4 +287,51 @@ function writeDNSPacket(parsed) {
   }));
 }
 
-module.exports = { types, classes, writeDNSPacket, parseDNSPacket };
+
+const addresses = {
+  // A generic host that has registered most DNS records
+  // and provides HTTP/HTTPS services, with both IPv4 and IPv6 support
+  INET_HOST: 'nodejs.org',
+  // A host that provides IPv6 services
+  INET4_HOST: 'nodejs.org',
+  // A host that provides IPv4 services
+  INET6_HOST: 'nodejs.org',
+  // An accessible IPv6 IP,
+  // defaults to the Google Public DNS IPv6 address
+  INET6_IP: '2001:4860:4860::8888',
+  // An accessible IPv4 IP,
+  // defaults to the Google Public DNS IPv4 address
+  INET4_IP: '8.8.8.8',
+  // An invalid host that cannot be resolved
+  // See https://tools.ietf.org/html/rfc2606#section-2
+  INVALID_HOST: 'something.invalid',
+  // A host with MX records registered
+  MX_HOST: 'nodejs.org',
+  // A host with SRV records registered
+  SRV_HOST: '_jabber._tcp.google.com',
+  // A host with PTR records registered
+  PTR_HOST: '8.8.8.8.in-addr.arpa',
+  // A host with NAPTR records registered
+  NAPTR_HOST: 'sip2sip.info',
+  // A host with SOA records registered
+  SOA_HOST: 'nodejs.org',
+  // A host with CNAME records registered
+  CNAME_HOST: 'blog.nodejs.org',
+  // A host with NS records registered
+  NS_HOST: 'nodejs.org',
+  // A host with TXT records registered
+  TXT_HOST: 'nodejs.org',
+  // An accessible IPv4 DNS server
+  DNS4_SERVER: '8.8.8.8'
+};
+
```

Comment:
I am not sure about the `locality` and `proximity` of our test systems w.r.t the system that hosts `http://nodejs.org` . If they are too close / under one subnet etc. how some of the tests (like dns resolution, pings, downloads etc.) are going to respond in terms of network level operations - will there be any bypasses?